package.json

```diff
@@ -8,7 +8,7 @@
     "boot": "yarn workspaces foreach -v -pA --topological-dev --no-private run build",
     "clean": "node scripts/clean.js",
     "storybook": "storybook dev -p 6006",
-    "storybook:aksel": "concurrently \"yarn watch:tw\" \"cross-env STORYBOOK_STORIES=all storybook dev -p 6006\"",
+    "storybook:aksel": "yarn concurrently \"yarn watch:tw\" \"yarn storybook dev -c aksel.nav.no/website/.storybook -p 6006\"",
```

Comment:
Burde de to storybookene kjøres på forskjellige porter?

Reply:
Fikset